lib/generators/boring/ci/gitlab_ci/install/install_generator.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Boring
+  module Ci
+    module GitlabCi
+      class InstallGenerator < Rails::Generators::Base
+        desc 'Adds Gitlab CI to the application'
+        source_root File.expand_path("templates", __dir__)
+
+        RUBY_VERSION_FILE = ".ruby-version"
+        DEFAULT_RUBY_VERSION = '.ruby-version'
+
+        class_option :ruby_version, aliases: '-v',
+                     desc: "Tell us the ruby version which you use for the application. Default to Ruby #{DEFAULT_RUBY_VERSION}, which will cause the action to use the version specified in the #{RUBY_VERSION_FILE} file."
```

Comment:
We can remove `, which will cause the action to use the version specified in the #{RUBY_VERSION_FILE} file.`